### DISCUSSION TITLE
Should we remove 'age death' ?

### DISCUSSION THREAD

**Discussion by rezalas** [rezalas/riftshadow#210] [Ideas]
Currently once your character reaches a specific age based on the date of it's creation, your character is killed and you become a ghost. After that, once you log out, you can no longer log in as that character. This actually has a number of benefits including eliminating the possibility of hording items, limits the lifespan of characters that are too powerful, and helps create lore for the game ("remember when" stories). However, with very few players it can be annoying to have a character die as well. Perhaps removing it, or at least making it adjustable so it could be turned off / on in the config settings.

**Comment by sean-gilliam** [comment]
We could certainly have a switch like that so server maintainers can set it up like they want.

Some more possibilities might include:
- A one-time quest chain to extend the character's life (permanently or by a set amount).
- A quest chain that, once a character reaches a certain level, they can pursue to reset their character to a "beginning" state but carry over certain items/stats/abilities from their former life (sorta like paragons from other games).
- If a player is gone for an extended time, then have them have the ability to "freeze" their character until they return. This could also be automated by the server.
- Have the option for that character to retire and become an NPC.

Just some ideas. :)

**Reply by rezalas** [reply]
I absolutely love these ideas! I could see all of these being great to be honest. We'd need to consider what someone gets for paragon. Perhaps even adding something permanent like an aura, title, etc. after you do it twice.

**Reply by Psypher9** [reply]
Retiring to be an NPC would be wicked cool. I think it would be fun to use the logs of that player's actions and generate conversation out of it for interaction

**Reply by Psypher9** [reply]
The Walt Disney option on number 3 is pretty cool too. I wonder, would the player cast a freeze self spell or something and just immediately get logged out?

**Reply by sean-gilliam** [reply]
That would be amazing if we could generate a script from the logs. Might be a bit to much for us. But with LLMs advancing the way they are, we could feed the logs into one and generate a script.

**Reply by sean-gilliam** [reply]
For freezing bit, yeah, I was thinking either a spell or potion. Maybe even a device of some sort, that when used would freeze them at the spot they used it and logged them off. Maybe give them a countdown to finish up what they were doing.
